ddpui/management/commands/patch_edr_deployment_env.py

```
@@ -0,0 +1,94 @@
+
+        for orgtask in qs:
+            org = orgtask.org
+            orgdbt = org.dbt
+            if orgdbt is None or orgdbt.dbt_profile_secret_block is None:
+                self.stdout.write(f"  [skip] {org.slug}: no dbt_profile_secret_block FK")
+                skipped += 1
+                continue
+
+            try:
+                dbt_project_params = DbtProjectManager.gather_dbt_project_params(org, orgdbt)
+            except Exception as err:  # pylint: disable=broad-exception-caught
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)